rel/app/env.sh.eex

```diff
@@ -2,10 +2,8 @@ if [ -f "$HOME/.livebookdesktop.sh" ]; then
   . "$HOME/.livebookdesktop.sh"
 fi
 
-hostname=`hostname`
-if [[ "$hostname" =~ " " ]]; then
-  echo "[error] system hostname ($hostname) cannot contain whitespaces"
-  exit 1
+if [[ "$LIVEBOOK_EPMDLESS" != "false" && "$LIVEBOOK_EPMDLESS" != "0" ]]; then
```

Comment:
I think we need to set the env var:

```suggestion
export LIVEBOOK_EPMDLESS=${LIVEBOOK_EPMDLESS:-false}

if [[ "$LIVEBOOK_EPMDLESS" != "false" && "$LIVEBOOK_EPMDLESS" != "0" ]]; then
```

so that in Elixir land this can be executed:

```elixir
if Livebook.Config.boolean!("LIVEBOOK_EPMDLESS", false) do
```